main/compute_gram_quadcopter.jl

```diff
@@ -0,0 +1,126 @@
+using FTCTests
+const FTC = FaultTolerantControl
+using LinearAlgebra
+using Plots
+
+
+"""
+# Refs
+[1] M. Tahavori and A. Hasan, “Fault recoverability for nonlinear systems with application to fault tolerant control of UAVs,” Aerosp. Sci. Technol., vol. 107, p. 106282, 2020, doi: 10.1016/j.ast.2020.106282.
+
+# Notes
+- x = state vector, [eta, omega, v, xi]^T
+    - eta = Euler angle vector, [phi, theta, psi]^T
+    - omega = angular rate vector, [p, q, r]^T
+    - v = velocity vector, [u, v, w]^T
+    - xi = position vector, [x, y, z]^T
+- u = control input vector, thrust vector, [omega_1, omega_2, omega_3, omega_4]^T
```

Comment:
```suggestion
- u = control input vector, thrust vector, [f1, f2, f3, f4]^T
```
실제 추력을 의미한다면, angular rate 과 안 헷갈리게 `f`가 낫겠습니다.

Reply:
논문의 노테이션을 따라가기 위해 이렇게 작성하였으나.. 수정하는 것이 나을까요?

Reply:
아하... 그렇군요. 조금 아쉽네요.

그럼 이건 @hnlee77 마음대로~

Reply:
일단 요건 고대로 둘게요~